### PR TITLE
Update file layout

### PR DESCRIPTION
**Pull request by doolio**
There should be no changes to any logic. I think these changes will prove useful though as I believe I have found some minor errors which I will submit in a new PR based on this one.

### CODE REVIEW

File: org-gtd-backward-compatibility.el [hunk 1]
```diff
@@ -23,10 +23,29 @@
...
+(defun org-gtd--ensure-list (object)
+  "Return OBJECT as a list.
+If OBJECT is already a list, return OBJECT itself.  If it's
+not a list, return a one-element list containing OBJECT."
+  (if (listp object)
+      object
+    (list object)))
+
+;; this was added in emacs 28.1
+(unless (fboundp 'ensure-list)
```
Comment (Trevoke) [inline org-gtd-backward-compatibility.el:44]: This is not private logic; this is actual logic executed when the file is loaded. How does this fit? Does this need a new comment?

File: org-gtd-areas-of-focus.el [hunk 2]
```diff
@@ -78,5 +73,23 @@ variable."
...
+(defun org-gtd-areas-of-focus--set ()
+  "Use as a hook when decorating items after clarifying them.
+
+This function requires that the user input find a match amongst the options.
+If a new area of focus pops up for you, change the value of the eponymous
+variable."
+  (unless (org-gtd-organize-type-member-p '(project-task trash knowledge quick-action))
+    (org-gtd-area-of-focus-set-on-item-at-point)))
+
+(defalias 'org-gtd-set-area-of-focus 'org-gtd-areas-of-focus--set)
```
[the remark this answers is not in the document]
Comment (doolio) [reply org-gtd-areas-of-focus.el:89]: OK, I kept the `defalias`es with the associated function for context but they can be moved to the public function subsection if we don't rename the private function. Let's consider doing that again outside of this PR.

**Comment by Trevoke** [comment]
I made the change to move the defalias to the public function area, I checked that it didn't create any linting errors and it didn't seem to. Thanks for doing all of this!

File: org-gtd-backward-compatibility.el (hunk 1 above)
Comment (Trevoke) [reply org-gtd-backward-compatibility.el:44]: Yeah, welcome to supporting multiple versions of emacs.

**Pull request merged**